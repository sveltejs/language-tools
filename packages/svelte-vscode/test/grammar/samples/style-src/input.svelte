<style src="../external.css"></style>
