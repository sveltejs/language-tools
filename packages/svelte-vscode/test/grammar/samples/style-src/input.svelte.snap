><style src="../external.css"></style>
#^ source.svelte meta.style.svelte meta.tag.start.svelte punctuation.definition.tag.begin.svelte
# ^^^^^ source.svelte meta.style.svelte meta.tag.start.svelte entity.name.tag.svelte
#      ^ source.svelte meta.style.svelte meta.tag.start.svelte
#       ^^^ source.svelte meta.style.svelte meta.tag.start.svelte meta.attribute.src.svelte entity.other.attribute-name.svelte
#          ^ source.svelte meta.style.svelte meta.tag.start.svelte meta.attribute.src.svelte punctuation.separator.key-value.svelte
#           ^ source.svelte meta.style.svelte meta.tag.start.svelte meta.attribute.src.svelte string.quoted.svelte punctuation.definition.string.begin.svelte
#            ^^^^^^^^^^^^^^^ source.svelte meta.style.svelte meta.tag.start.svelte meta.attribute.src.svelte string.quoted.svelte
#                           ^ source.svelte meta.style.svelte meta.tag.start.svelte meta.attribute.src.svelte string.quoted.svelte punctuation.definition.string.end.svelte
#                            ^ source.svelte meta.style.svelte meta.tag.start.svelte punctuation.definition.tag.end.svelte
#                             ^^ source.svelte meta.style.svelte meta.tag.end.svelte punctuation.definition.tag.begin.svelte
#                               ^^^^^ source.svelte meta.style.svelte meta.tag.end.svelte entity.name.tag.svelte
#                                    ^ source.svelte meta.style.svelte meta.tag.end.svelte punctuation.definition.tag.end.svelte
>